taskcluster/ci/nightly-verify/kind.yml

```
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+        
+    test:
+        description: Run a simple UI test to verify Nightly build can start 
+        worker-type: b-android
+        attributes:
+            build-type: debug
+            code-review: true
+        run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            using: run-commands
+            use-caches: false
+            commands:
+                - [wget, {artifact-reference: '<signing/public/build/x86/geckoNightly/target.apk>'}, '-O', app-x86.apk]
+                - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', app-arm.apk]
+                - [wget, {artifact-reference: '<signing-android-test/public/build/noarch/geckoNightly/target.apk>'}, '-O', android-test.apk]
+                - [automation/taskcluster/androidTest/ui-test.sh, x86-start-test, app-x86.apk, android-test.apk, '-1']
+                - [automation/taskcluster/androidTest/ui-test.sh, arm-start-test, app-arm.apk, android-test.apk, '-1']
+            secrets:
+                - name: project/mobile/fenix/firebase
+                  key: firebaseToken
+                  path: .firebase_token.json
+                  json: true
+        treeherder:
+            kind: test
+            platform: 'nightly-start-test/opt'
+            symbol: debug(ui-test-x86)
+            tier: 2
+        worker:
+            docker-image: {in-tree: ui-tests}
+            max-run-time: 7200
+            env:
+                GOOGLE_APPLICATION_CREDENTIALS: '.firebase_token.json'
+                GOOGLE_PROJECT: moz-fenix
+            artifacts:
+                - name: public
+                  path: /build/fenix/results
+                  type: directory
+        dependencies:
+            signing: signing-debug 
+            signing-android-test: signing-android-test
```

Bug: Ah, I just found a bug. The fact that we need `android-test` to have the same signing key as nightly requires  some changes in the signing kind. I'm taking care of it today.

---

TODO before landing:

```suggestion
            signing-android-test: signing-android-test-nightly
```